classes/game.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'item'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
```

Comment:
- Based on UML, class Game should inherit from class Item the method: add_game. Please add it.
- I suggest a game should have a title or name for itself.